### PR TITLE
add request accept page

### PR DESCRIPTION
**Pull request by gabimartini**
Add the UI for request-accepted page.
In the received-requests page there's a ID with link for the completed request

The data is hard coded
Screenshot 2023-09-04 at 12 01 59

### CODE REVIEW

File: src/views/supplier/request-accepted.njk [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    <h2 class="govuk-heading-m govuk-!-static-margin-bottom-1">Feedback provided</h2>
+    <p class="govuk-body">We are in a good place to start the MoU but will need to set up a meeting to discuss the finer details.</p>
+
+    <h2 class="govuk-heading-m govuk-!-static-margin-bottom-1">Notes</h2>
+    <p class="govuk-body">I spoke to George Heatherfield from legal who confirmed the correct legal gateway.</p>
+
+
+    <h2 class="govuk-heading-m govuk-!-static-margin-bottom-1">What happens next</h2>
+    <p class="govuk-body">The Department for Education have been told that this request was accepted.</p>
+    <p class="govuk-body">Contact <a href="#1">data-management@dfe.gov.uk</a> to set up the data share arrangement.</p>
```
Comment (madz-lw) [inline src/views/supplier/request-accepted.njk:27]:
```suggestion
    <p class="govuk-body">Contact <a class="govuk-link" href="#1">data-management@dfe.gov.uk</a> to set up the data share arrangement.</p>
```

Thanks Gabi 🙂 I would just add the govuk-link class to the email anchor tag but other than that looks good 🎉

**Review by madz-lw — CHANGES_REQUESTED** [review]

File: src/views/supplier/request-accepted.njk (hunk 1 above)
Comment (gabimartini) [reply src/views/supplier/request-accepted.njk]: Thanks @madz-lw I've added now :D

**Review by madz-lw — APPROVED** [review]
Thanks Gabi 🎉

**Review by AShaww — APPROVED** [review]
LGTM, thanks Gabi 👍🏼

**Pull request merged**